Generation/src/components/HepMCFileWriter.h

```diff
@@ -0,0 +1,35 @@
+#ifndef GENERATION_HEPMCFILEWRITER_H
+#define GENERATION_HEPMCFILEWRITER_H
+
+
+#include "GaudiAlg/GaudiAlgorithm.h"
+
+#include "FWCore/DataHandle.h"
+
+namespace HepMC {
+  class GenEvent;
+  class IO_GenEvent;
+}
+
+class HepMCFileWriter: public GaudiAlgorithm {
```

Comment:
Sure, but then I think we should lead by example and stop storing store our samples in this format ;-)